src/components/ServiceDetails/AboutService/DemoExample/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from "react";
+import { withStyles } from "@material-ui/styles";
+
+import ProgressBar from "../../../common/ProgressBar";
+import { useStyles } from "./styles";
+import ServiceProvider from "./ServiceProvider";
+
+class DemoExample extends Component {
```

Comment:
Could be renamed as `ServiceDemo`.

Reply:
[renamed](https://github.com/singnet/refactored-dapp/pull/13/commits/ea4add782480235c55f8ca04d7fcab3065fe31fd)